player/main.c

```diff
@@ -427,6 +427,10 @@ int mpv_main(int argc, char *argv[])
 
     char **options = argv && argv[0] ? argv + 1 : NULL; // skips program name
     int r = mp_initialize(mpctx, options);
+#if !defined(__SANITIZE_ADDRESS__)
+    // allow fast exit on standalone player, unless leak detection is enabled
+    mpctx->quit_fast = !getenv("MPV_LEAK_REPORT");
```

Comment:
The actual `MPV_LEAK_REPORT` logic is a bit more complicated that involves handling "0" and `HAVE_TA_LEAK_REPORT` as well.

But I don't think it's worth bothering with here. Just pointing it out.

Reply:
oh god, why do we need two ways to enable one thing. What's wrong with env variable.

Reply:
done